issue_493.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
Not sure if this is still in "work in progress" mode, but if not, can this script get removed?

Reply:
removing..